ui/js/admin.js

```diff
@@ -498,7 +499,56 @@ jQuery( function( $ ) {
 			}
 		});
 	});
+	var updateFilterActions = function( element ) {
+		var connector = 'Any';
+		var connector_element = $( 'select[name="context"]' );
+		var select2_data = connector_element.select2('data');
+		if ( undefined !== select2_data[0].element['attributes']['data-group'] ) {
```

Comment:
Can this be alternately expressed as:
`select2_data[0].data( 'group' )`?

Or maybe:
`select2_data[0].attr( 'data-group' )`?